nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -45,6 +45,8 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         <CppWinRT_IncludePath>$(GeneratedFilesDir)</CppWinRT_IncludePath>
         <!--TEMP: Override NuGet SDK's erroneous setting in uap.props -->
         <WindowsSDK_MetadataFoundationPath Condition="('$(WindowsSDK_MetadataFoundationPath)'!='') And !Exists($(WindowsSDK_MetadataFoundationPath))">$(WindowsSDK_MetadataPathVersioned)</WindowsSDK_MetadataFoundationPath>
+        <!-- CAExcludePath is used to set an environment variable, so make sure this is defined on a single line. -->
+        <CAExcludePath>$(GeneratedFilesDir);$(CAExcludePath)</CAExcludePath>
```

Comment:
The position of the XML element changed. Does that matter?

The order within the include path also changed. Now GeneratedFilesDir comes first. Does that matter?

Reply:
I updated the order to match how the system props files add path to the variable. Shouldn't matter. The location is practically the same. All properties below the old and new location are used as the input to DependsOnTargets so the order doesn't matter. It is also not used in between.